Makefile.in

```diff
@@ -54,9 +56,11 @@ configure config.h.in: configure.ac
 	./autogen.sh
 
 install: all
+	$(MKDIR) -p $(DESTDIR)$(bindir)
 	$(MKDIR) -p $(DESTDIR)$(sbindir)
 	$(MKDIR) -p $(DESTDIR)$(mandir)/man1
-	$(INSTALL) -m 0755 $(PROG) $(DESTDIR)$(sbindir)
+	$(INSTALL) -m 0755 $(PROG) $(DESTDIR)$(bindir)
+	$(INSTALL) -l as $(DESTDIR)$(bindir)/$(PROG) $(DESTDIR)$(sbindir)/$(PROG)
```

Comment:
Alas, it seems that `install -l` is not portable. GNU `install` doesn't seem to provide similar functionality. Maybe just resort to directly using `ln -s` instead?

Reply:
Good catch, I have BSD userland. I dare to remember that I have seeen `$(LN_S}` somewhere and autoconf will try `${INSTALL -l)` or `ln -s` directly.

Reply:
Here it is: https://github.com/cyrusimap/cyrus-imapd/blob/7fe7c364c22e70acadfe70e7e5915eb20676c550/configure.ac#L135

Reply:
@dkobras  Moved to a default macro, have a look.